packages/perseus/src/widgets/label-image/answer-pill.tsx

```diff
@@ -8,16 +8,31 @@ import {Popper} from "react-popper";
 
 import Renderer from "../../renderer";
 
+const BringToFront: CSSProperties = {
```

Comment:
**question**: Can this be moved to the `styles` object below?

Reply:
The styles object doesn't take variables

Reply:
Weirdly, it doesn't work when it's there.